keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -806,7 +806,7 @@ checks for events that describe the problem.
 - 5 installs
 - More rigorous forms of testing e.g., downgrade tests and scalability tests
 - Allowing time for feedback
-- Integration with [Cluster Autoscaler](https://github.com/kubernetes/autoscaler)
+- Design for support in [Cluster Autoscaler](https://github.com/kubernetes/autoscaler)
```

Comment:
This is a slightly relaxed criteria: https://github.com/kubernetes/autoscaler/pull/3887 shows that the current in-tree API is sufficient to enable autoscaling, the PR just hasn't been merged yet because SIG Autoscaling wanted more time to investigate how this can be made simpler for users.

The recommendation from the SIG Autoscaling on 2022-02-21 was to not wait for that.

Reply:
Can you add a link to this PR in the KEP to show this is in progress?

Reply:
Can you update [this section](https://github.com/pohly/enhancements/blob/master/keps/sig-storage/1472-storage-capacity-tracking/README.md#integration-with-cluster-autoscaler) with a summary of the design?

Reply:
FYI @x13n @MaciekPytel

Reply:
Will do. It basically works exactly as alluded in that section: labeling of generated nodes must be modified to distinguish them from real ones and then manually created CSIStorageCapacity objects provide the information about those future nodes.